components/clickup/actions/common/constants.mjs

```diff
@@ -0,0 +1,6 @@
+export const PRIORITIES = {
+  "Urgent": 1,
+  "High": 2,
+  "Normal": 3,
+  "Low": 4,
+};
```

Comment:
I got an error while testing saying that `constants.mjs` doesn't have an `export default`. I think it should be written this way:
```suggestion
export default {
  PRIORITIES: {
    "Urgent": 1,
    "High": 2,
    "Normal": 3,
    "Low": 4,
  }
};
```